CMSgov/design-system
# [WNMGDS-2402] Migrate all prop tables from doc site

## Summary

Second PR for WNMGDS-2402

- Creates a component capable of linking to multiple sets of storybook docs for redirecting dev users to storybook
  - Uses that component for pages that correspond to multiple storybook doc pages
- Removes prop table components in the Gatsby site
- Removes infrastructure for generating the prop tables

## How to test

1. Start the doc site
2. Look through the component pages to make sure all is well

### If this is a change to documentation:
